cdap-runtime-ext-remote-hadoop/src/main/java/io/cdap/cdap/runtime/spi/provisioner/remote/RemoteHadoopConf.java

```diff
@@ -92,7 +90,7 @@ public static RemoteHadoopConf fromProperties(Map<String, String> properties) {
     SSHKeyPair keyPair = new SSHKeyPair(new SSHPublicKey(user, ""),
                                         () -> privateKey.getBytes(StandardCharsets.UTF_8));
 
-    EdgeNodeCheckType edgeNodeCheckType = EdgeNodeCheckType.getEdgeNodeCheck(properties.get("edgeNodeCheckMethod"));
+    EdgeNodeCheckType edgeNodeCheckType = EdgeNodeCheckType.fromString(properties.get("edgeNodeCheckMethod"));
     int checkTimeout = parseCheckTimeout(properties.get("checkTimeout"));
```

Comment:
Can we store properties names as constants?

Reply:
Made all property names constant.